includes/fs-essential-functions.php

```diff
@@ -292,15 +292,25 @@ function fs_find_caller_plugin_file() {
 	function fs_update_sdk_newest_version( $sdk_relative_path, $plugin_file = false ) {
 		global $fs_active_plugins;
 
+		$newest_sdk = $fs_active_plugins->plugins[ $sdk_relative_path ];
```

Comment:
@fajardoleo About all this function - isn't the fact that you changed risks the whole `start.php` logic? Because when a theme with the new SDK will be activated for the first time it will load the old of this function (`fs_update_sdk_newest_version()`)? 

I didn't check/test it, it just something suspicious that I spotted.